beheer.md

```diff
@@ -104,13 +104,17 @@ docker service ps getstartedlab_web | Haal de gegevens op van alle tasks die de
 docker stack rm getstartedlab | Afsluiten van de applicatie.
 docker swarm leave --force | Afsluiten van de swarm.
 
-**Criteria**
-
-* API specificatie (OAS3) moet beschikbaar zijn en uit de imvertor genereerbaar zijn
-* Documentatie (zowel technisch als functioneel) moet beschikbaar zijn
-* De referentie-implementatie moet gereed en up-to date zijn met dan geldende OAS3
-* Het gebruikte Informatiemodel (minimaal UML)
-* Geautomatiseerde test en buildstraat (CI) moet beschikbaar voor de RI
-* De standaard moet ingebouwd zijn bij leveranciers (3 of meerdere leveranciers)
-* De standaard bevat een tabel waarin wordt aangegeven welke API versies met welke andere API versies compatible zijn.
+##Criteria voor in beheername van API standaarden
+API standaarden dienen aan de hieronder genoemde criteria te voldoen voordat ze door beheer in beheer genomen kunnen worden.
+
+- De standaard moet formeel goedgekeurd zijn.
+>_Ik ga er vanuit dat er nog een governance document komt waarin staat hoe en door wie een API standaard wordt goedgekeurd._
+- API specificatie (OAS3) moet beschikbaar zijn en uit de imvertor genereerbaar zijn. Dit betekent dat zowel de Semantische Informatiemodellen (SIM), Uitwisselings Gegevensmodellen (UGM) als het Bericht Specificatiemodel (BSM) in UML beschikbaar moeten zijn.
```

Comment:
```suggestion
- API specificatie (OAS3) moet beschikbaar zijn en uit de imvertor genereerbaar zijn. Dit betekent dat zowel de Semantische Informatiemodellen (SIM), Uitwisselings Gegevensmodellen (UGM) als het Bericht Specificatiemodel (BSM) in UML beschikbaar moeten zijn.
_Dit leidt bij enkelen nog tot de volgende vragen:
Van wie moet dit?
Voor wie moet dit?
Welk doel dient het?_
```

Reply:
Ik denk dat we met deze tijdelijke toevoegingen deze PR moeten kunnen accepteren zodat we verder kunnen met het vullen va dit bestand. De opmerkingen komen zo echter niet te vervallen.

Reply:
Sorry maar met die toevoeging keur ik het niet goed. In mijn ogen moet deze eis er uit, en in een aparte User Story worden behandeld. Zo kan deze PR doorgang vinden, maar dan zonder deze eis die dan toch op de radar blijft.

Het betreft hier niet een nuancering, aanvulling of kleine wijziging. Dit is een breek-punt voor de ZGW APIs waar API-beheer mee zou gaan starten. Ik zie namelijk de User Story om met Imvertor te werken nergens terug en dit kwam vrij random uit de lucht vallen.

Reply:
Ik stel voor dit te wijzigen tot iets als:
API specificatie (OAS3) moet beschikbaar zijn en in de documentatie moeten UML modellen van de in de API beschikbare resources en de onderlinge relaties beschikbaar zijn.

Dat is dus waar Remko nu mee bezig is in de functionele documentatie.

Dit sluit ook aan bij hoofdstuk drie Gebruikerswensen van de API Strategie: https://geonovum.github.io/KP-APIs/#gebruikerswensen

Reply:
@michielverhoef Dat klinkt veel beter en is ook resultaat/product gericht en niet procesgericht (ik zie Imvertor als een proces om tot een OAS te komen)

Reply:
Ik zie het iets anders: Imvertor is een stuk gereedschap wat je kunt gebruiken om vanuit een UML model, gemaakt in Enterprise Architect, een OAS3 file te genereren. Of StUF schema's (als je wilt ;-) ).

Het is dus niets meer of minder dan een plugin. Wel eentje met veel mogelijkheden maar niet meer dan dat.

Reply:
Ik heb deze eis inmiddels zo aangepast zodat het een weerspiegeling is van het besluit dat op donderdag 21-2-2019 is genomen in dit kader.